### PR TITLE
👀 monitor visitor sign in type

### LINKED ISSUE #226: 👀 monitor sign in type for visitor apps
Monitor whether visitors are using sign in with name or sign in with QR code.

@eliascodes do you want to roll this in with #207?

### PR DESCRIPTION
**Pull request by astroash**
**NOTE:** Dates for migration may need editing depending on whether this PR or #227 goes in first

fixes #226

### Changes
- create table to track visitor sign in type
- visitor app req specifies sign in type
- store sign in type to db

### Testing Requirements
- [x] Visitor App
  - log in visitor with QR code
  - log in visitor with sign in with name
  - manually check db is updates (optional)

### Release Requirements
- no restrictions. 
- deploy when ready.

### Manual Deployment
- [ ] Visitor App
- [ ] API

### CODE REVIEW

File: api/src/api/v1/community_businesses/visit_logs.ts [hunk 1]
```diff
@@ -13,10 +14,13 @@ import { ApiRequestQuery } from '../schema/request';
 import Roles from '../../../models/role';
 import { RoleEnum } from '../../../models/types';
 
-interface VisitorSearchRequest extends Hapi.Request {
+export type VisitsignInType = 'sign_in_with_name' | 'qr_code';
```
Comment (eliascodes) [inline api/src/api/v1/community_businesses/visit_logs.ts]:
```suggestion
export type VisitSignInType = 'sign_in_with_name' | 'qr_code';
```

File: visitor-app/src/visitors/pages/SignIn/index.js [hunk 2]
```diff
@@ -82,11 +83,13 @@ export default class SignIn extends Component {
   }
 
   onSignInSuccess = (res) => {
+    console.log({ res });
```
Comment (eliascodes) [inline visitor-app/src/visitors/pages/SignIn/index.js]: Can we get rid of this?

File: visitor-app/src/visitors/pages/SignIn/index.js [hunk 3]
```diff
@@ -118,9 +122,11 @@ export default class SignIn extends Component {
       this.props.history.push('/error/unknown');
     }
 
+    console.log(this.state);
```
Comment (eliascodes) [inline visitor-app/src/visitors/pages/SignIn/index.js]: And this?

**Comment by astroash** [comment]
@eliascodes should be gone now

**Pull request merged**